forge-ai/src/main/java/forge/ai/ability/TokenAi.java

```diff
@@ -202,11 +202,11 @@ protected boolean checkApiLogic(final Player ai, final SpellAbility sa) {
 
         if (sa.isPwAbility() && alwaysFromPW) {
             return true;
-        } else if (game.getPhaseHandler().is(PhaseType.COMBAT_DECLARE_ATTACKERS)
+        } else if (game.getCombat() != null
```

Comment:
why this change?

Reply:
Ah, that was a remainder from an experiment, good catch, thanks! :)